Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -126,6 +126,10 @@ function setUpAlert() {
   }
 }
 
+function setUpSymbol() {
+  GLOBAL.Symbol = require('es6-symbol');
```

Comment:
https://github.com/medikoo/es6-symbol/blob/master/index.js#L3

Am I reading that wrong? It seems like it's checking for native, global `Symbol` before it polyfills.

Or would you rather have that check in our code?